internal/cli/commands_test.go

```diff
@@ -1,80 +1,249 @@
 package cli
 
 import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
 	"testing"
+	"time"
+
+	"github.com/stretchr/testify/suite"
 )
 
-func TestCleanDir(t *testing.T) {
+type CreateCmdSuite struct {
+	suite.Suite
+}
+
+func TestCreateCmdSuite(t *testing.T) {
+	suite.Run(t, &CreateCmdSuite{})
+}
+
+func (s *CreateCmdSuite) mustCreateTempDir() string {
+	tmpDir, err := ioutil.TempDir("", "")
```

Comment:
NVM, using `afero` will probably be too much work. e.g. we'd need to create files in `createCmd()` using a filesystem interface instead of using the `os` package directly